### ISSUE TITLE
Testing 

### ISSUE THREAD

**Issue by weird-budy** [sosauce/CuteMusic#14]
Looks like your app is not tested on Samsung devices. I'm also a curious teenager like you. So if you say i would like to do that

**Comment by sosauce** [comment]
Sure ! I take testing from all devices, crucial since I can't reproduce some bigs, notably one that occurs on OxygenOS

**Comment by weird-budy** [comment]
Ok i have already installed it. I will test it . let me see if i find anything worng with it

**Comment by weird-budy** [comment]
@sosauce this is the update on testing cutemusic on Samsung device . I tested it on one ui 5.1 with Android 13. After using it for 2 days I found a problem. 
Looks like when you play a songs with unknown artist it plays the songs without any problem but doesn't show the name of the song in mini player or the main player. The problem stays even if you play another song .
The other one is when you play a song with artist name there is no problem but when you change the song it doesn't change the name. Here is a screen recording to clear everything 

https://github.com/sosauce/CuteMusic/assets/154891033/764d20c3-1b67-4263-953f-f68fb9d59034
